bioblend/galaxy/invocations/__init__.py

```diff
@@ -247,8 +250,33 @@ def get_invocation_biocompute_object(self, invocation_id):
         url = self._make_url(invocation_id) + '/biocompute'
         return self._get(url=url)
 
+    def wait_for_invocation(self, invocation, timeout_seconds=10):
+        """
+        Wait for an invocation to be scheduled.
+
+        :type invocation: dict
+        :param invocation: Invocation to wait for.
+
+        :type timeout_seconds: int
+        :param timeout_seconds: Timeout in seconds. If the invocation is not scheduled
+                                after this timeout, an InvocationNotScheduledException
+                                is raised.
+        """
+        for _ in range(timeout_seconds * 2):
+            invocation = self.gi.workflows.show_invocation(invocation['workflow_id'], invocation['id'])
+            if invocation['state'] in INVOCATION_TERMINAL_STATES:
+                break
+            time.sleep(.5)
+        invocation = self.gi.workflows.show_invocation(invocation['workflow_id'], invocation['id'])
```

Comment:
```suggestion
```

Reply:
Woudn't it then raise an exception about waiting for `n` seconds while actually only checking for `n - 0.5` seconds?

Reply:
I see what you mean, let's leave it like this then